docs/handlers.md

```diff
@@ -0,0 +1,1451 @@
+| Format        | Type                                 | Fully supported?    |
+| :------------ | :----------------------------------- | :-----------------: |
+| [`AR`](#ar) | ARCHIVE | :octicons-check-16: |
+| [`ARC`](#arc) | ARCHIVE | :octicons-check-16: |
+| [`ARJ`](#arj) | ARCHIVE | :octicons-check-16: |
+| [`BDL`](#bdl) | ARCHIVE | :octicons-check-16: |
+| [`BNEG`](#bneg) | ARCHIVE | :octicons-check-16: |
+| [`BZIP2`](#bzip2) | COMPRESSION | :octicons-check-16: |
+| [`CAB`](#cab) | ARCHIVE | :octicons-check-16: |
+| [`CHK`](#chk) | ARCHIVE | :octicons-check-16: |
+| [`COMPRESS`](#compress) | COMPRESSION | :octicons-check-16: |
+| [`CPIO_BINARY`](#cpio_binary) | ARCHIVE | :octicons-check-16: |
+| [`CPIO_PORTABLE_ASCII`](#cpio_portable_ascii) | ARCHIVE | :octicons-check-16: |
+| [`CPIO_PORTABLE_ASCII_CRC`](#cpio_portable_ascii_crc) | ARCHIVE | :octicons-check-16: |
+| [`CPIO_PORTABLE_OLD_ASCII`](#cpio_portable_old_ascii) | ARCHIVE | :octicons-check-16: |
+| [`CRAMFS`](#cramfs) | FILESYSTEM | :octicons-check-16: |
+| [`DMG`](#dmg) | ARCHIVE | :octicons-check-16: |
+| [`ECC`](#ecc) | ARCHIVE | :octicons-check-16: |
```

Comment:
ECC has a meaning different what this format implies, it should be extended to autel_ecc, or similar.

Reply:
Fixed. I also improved the name of all vendor handlers (e.g. `bdl` -> `HP BDL`, `trx_v1` -> `Netgear TRX v1`).